src/Frontend/Themes/Bootstrap/src/Layout/Templates/Alerts.html.twig

```diff
@@ -1,7 +1,7 @@
 {% macro alert(type, message, dismissable, id) %}
   <div{% if id %} id="{{ id }}"{% endif %} class="alert alert-{{ type }}" role="alert">
     {% if dismissable %}
-      <button type="button" class="close" data-bs-dismiss="alert">×</button>
+      <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="{{ 'lbl.Close'|trans }}"></button>
```

Comment:
Misschien het label ook in content zelf steken? Of in title-attribute? Zodat het on over ook verschijnt?

Reply:
fixed in 39e5982